gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -325,6 +354,29 @@ public void delete_throwsExceptionWhenHadoopPathNull() throws IOException {
     assertThat(exception).hasMessageThat().startsWith("hadoopPath must not be null");
   }
 
+  @Test
+  public void open_IOstatistics() throws IOException {
```

Comment:
Is there a way to validate (manually) that Spark picks up the IOStatistics integrations ? If so, can we add some details in the description of the PR

Reply:
We will check